Resources/ProtobufDumper/ProtobufDumper/Program.cs

```diff
@@ -16,11 +19,66 @@ static void Main( string[] args )
                 return;
             }
 
-            var target = args[ 0 ];
-            var output = args.Length > 1 ? args[ 1 ] : null;
+            var targets = new List<String>();
+            string output = null;
 
-            var imgFile = new ImageFile( target, output );
-            imgFile.Process();
+            for ( var i = 0; i < args.Length; i++ )

```

Comment:
Won't this argument processing consume `-dump` below as a target or as the output dir?

Reply:
Split into named and unnamed arguments.